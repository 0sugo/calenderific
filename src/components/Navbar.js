import { NavLink } from 'react-router-dom';
import { LeftChev } from './Icons';
import globalphoto from './assets/globalphoto.png';

const Navbar = () => (
  <>
    <div className="nav-container">
      <NavLink to="/" className="back"><LeftChev /></NavLink>
      <h1 className="topper">World Holidays</h1>
    </div>
    <div>
      <img className="header-photo" src={globalphoto} alt="world" />
    </div>
  </>
);

export default Navbar;
