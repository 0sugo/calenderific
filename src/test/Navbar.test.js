import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from '../components/Navbar';

test('renders the Navbar component correctly', () => {
  const { container } = render(
    <Router>
      <Navbar />
    </Router>
  );

  expect(container).toMatchSnapshot();
});
