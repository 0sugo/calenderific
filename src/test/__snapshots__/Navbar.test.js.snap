// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Navbar component correctly 1`] = `
<div>
  <div
    class="nav-container"
  >
    <a
      aria-current="page"
      class="back active"
      href="/"
    >
      <svg
        class="w-6 h-6 leftchev"
        fill="none"
        stroke="currentColor"
        stroke-width="1.5"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M15.75 19.5L8.25 12l7.5-7.5"
          stroke-linecap="round"
          stroke-linejoin="round"
        />
      </svg>
    </a>
    <h1
      class="topper"
    >
      World Holidays
    </h1>
  </div>
</div>
`;
